Hestia/Hestia/frontend/EntryScreen/UIViewControllerLocalGlobal.cs

```diff
@@ -149,12 +154,12 @@ private void ToLocalScreen()
             else
             {
                 Console.WriteLine("To Server Connect screen");
-                UIStoryboard devicesMainStoryboard = UIStoryboard.FromName("Devices2", null);
+                UIStoryboard devicesMainStoryboard = UIStoryboard.FromName(strings.devices2StoryBoard, null);
                 PresentViewController(devicesMainStoryboard.InstantiateInitialViewController(), true, null);
             }
         }
 
-        private async Task ToGlobalScreen()
+        async Task ToGlobalScreen()

```

Comment:
Why make them public?

Reply:
The default is private. Visual studio for Mac flags the private as redundant